packages/flutter/test/material/text_field_test.dart

```diff
@@ -212,6 +212,151 @@ void main() {
     );
   }
 
+  testWidgets('text field selection toolbar should hide when the user starts typing', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: SizedBox(
+              width: 100,
+              height: 100,
+              child: Opacity(
```

Comment:
Why are you including this Opacity widget?

Reply:
Oh, indeed unnecessary, just copy from another test.